### ISSUE TITLE
Git clone error - fatal: Out of memory, malloc failed (tried to allocate 100000000000 bytes)

### ISSUE THREAD

**Issue by djaym7** [facebookresearch/ParlAI#4573]
What is the recommended config for git config ?

 git clone https://github.com/facebookresearch/ParlAI.git
Cloning into 'ParlAI'...
fatal: Out of memory, malloc failed (tried to allocate 100000000000 bytes)

**Comment by klshuster** [comment]
this looks like an issue with your machine - do you have enough disk space to download?

**Comment by stephenroller** [comment]
Uhhh our repo is definitely not 100gb. This is a local thing for sure.

**Comment by djaym7** [comment]
Yes, I have >4TB space available, this isn’t an issue with other git repos though, I tried few google/stackoverflow remedies but didn’t work like gitconfig hacks.

**Comment by djaym7** [comment]
Closing the issue, error went away somehow after restarting haha
